.github/workflows/update-oam-defs.yml

```
@@ -0,0 +1,104 @@
+      - name: Build jsonschema util
+        run: |
+          curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.38.0/install.sh | bash
+          export NVM_DIR="$HOME/.nvm"
+          [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"  # This loads nvm
+          nvm install v14.15.3
+          nvm install-latest-npm
+          git clone https://github.com/layer5io/kubeopenapi-jsonschema util
+          cd util
+          npm i; npm i nexe -g
+          make linux
+      - name: Generate Workload definitions
+        run: |
+          dirPath=../templates/oam/workloads/workloads-$NEW_OSM_VERSION
+          cd util
+          mkdir -p $dirPath
+          meshName=osm
+          smpMeshName=OPEN_SERVICE_MESH
+          template='{"apiVersion":"core.oam.dev/v1alpha1","kind":"WorkloadDefinition","metadata":{},"spec":{"definitionRef":{},"metadata":{"@type":"pattern.meshery.io/mesh/workload","meshVersion":"'$NEW_OSM_VERSION'","meshName":"'$smpMeshName'","k8sAPIVersion":null,"k8sKind":""}}}'
+          crds=$(./kubeopenapi-jsonschema --location ~/osm.yaml -t yaml --filter '$[?(@.kind=="CustomResourceDefinition")]' -o json --o-filter '$..["spec"]["names"]["kind"]' | jq '.[]')
+          for t in ${crds[@]}; do
+            nameUpper=`echo $t  | tr -d '"'`
+            nameLower=`echo $t  | tr -d '"' | tr '[:upper:]' '[:lower:]'`
+            definitionRef=$(printf %s.osm.meshery.layer5.io $nameLower)
+            apiVersion=$(./kubeopenapi-jsonschema --location ~/osm.yaml -t yaml --filter '$[?(@.kind=="CustomResourceDefinition" && @.spec.names.kind=='$t')]..spec.versions[0]' --o-filter "$[].name" -o json | jq '.[]' | tr -d '"')
+            apiGroup=$(./kubeopenapi-jsonschema --location ~/osm.yaml -t yaml --filter '$[?(@.kind=="CustomResourceDefinition" && @.spec.names.kind=='$t')]..spec' --o-filter "$[].group" -o json | jq '.[]' | tr -d '"')
+            ./kubeopenapi-jsonschema --location ~/osm.yaml -t yaml --filter '$[?(@.kind=="CustomResourceDefinition" && @.spec.names.kind=='$t')]..openAPIV3Schema.properties.spec' --o-filter "$[]" -o json |\
+            jq '
+            .[]
+            | .version = "'$NEW_OSM_VERSION'"
+            | ."object-type" = "'$nameUpper'"
+            | ."service-mesh" = "'$meshName'" ' > $dirPath/$nameLower.osm.meshery.layer5io.schema.json
+            echo $template |\
+            jq '
+            ."metadata"."name" = "'$(printf %s.OSM $nameUpper)'"
+            | ."spec"."metadata"."k8sAPIVersion" = "'$(printf $apiGroup/$apiVersion $apiGroup $apiVersion)'"
+            | ."spec"."metadata"."k8sKind" = "'$nameUpper'"
+            | ."spec"."definitionRef"."name"="'$definitionRef'"' > $dirPath/$nameLower.osm_definition.json
+          done
+      - name: Cleanup
+        run: |
+          rm -rf util
+      - name: Commit changes
+        uses: stefanzweifel/git-auto-commit-action@v4
+        with:
+          file_pattern: ./templates ./.github
```

```suggestion
          file_pattern: templates/
```
We don't need to add anything to `.github` now.